modules/effects/migrations/18_0_0/index.spec.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  SchematicTestRunner,
+  UnitTestTree,
+} from '@angular-devkit/schematics/testing';
+import { createWorkspace } from '@ngrx/schematics-core/testing';
+import { tags } from '@angular-devkit/core';
+import * as path from 'path';
+
+describe('Effects Migration to 18.0.0', () => {
+  const collectionPath = path.join(__dirname, '../migration.json');
+  const schematicRunner = new SchematicTestRunner('schematics', collectionPath);
+
+  let appTree: UnitTestTree;
+
+  beforeEach(async () => {
+    appTree = await createWorkspace(schematicRunner, appTree);
+  });
+
+  describe('replacements', () => {
+    const testParams = [
```

Comment:
What is the result of the migration if there's already an `@ngrx/operators` import statement?
I think it will create a new one, but I think it would be better to add the `concatLatestFrom` operator to the existing imports. 
On the other hand, this won't most likely be an issue.

Thoughts?

Reply:
@timdeschryver adding it to an existing import declaration should now also work